cmd/up/up.go

```diff
@@ -252,24 +252,36 @@ func (up *upContext) start(autoDeploy, build bool) error {
 // activateLoop activates the development container in a retry loop
 func (up *upContext) activateLoop(autoDeploy, build bool) {
 	isRetry := false
+	networkRetries := 0
 	for {
 		err := up.activate(isRetry, autoDeploy, build)
 		if err != nil {
 			if err == errors.ErrLostSyncthing {
 				isRetry = true
 				continue
-			} else {
-				up.Exit <- err
-				return
 			}
+
+			if errors.IsTransient(err) {
+				networkRetries++
```

Comment:
when do you think we should reset this counter? as is, it will do up to 3 times in the entire lifetime of the up. I think it would be better if we reset to 0 after a good connection.

Reply:
I would retry forever, and add a warning message telling okteto is waiting for network connectivity. Most app reconnects as soon as network is available

Reply:
Sounds good. Let's put a message on every retry cycle so it doesn't look like it's just frozen.